packages/i/imagemagick/xmake.lua

```diff
@@ -0,0 +1,25 @@
+package("imagemagick")
+    set_homepage("https://imagemagick.org/script/index.php")
+    set_description("ImageMagick is a FOSS software suite for modifying images. This does NOT provide any of the utilities. It installs the C/C++ Libraries.")
+    add_urls("https://download.imagemagick.org/ImageMagick/download/ImageMagick-$(version).tar.gz")
+    add_versions("7.0.11-13", "6c162d7cbd7b80968a6d083d39eb18d9c9bbd49f500f7f49c9a5dcc0fc36a03b")
+    add_deps("pkg-config")
+    add_deps("ffmpeg", "fftw", "jasper", "libjpeg-turbo", "libpng", "libtiff", "libwebp", "openjpeg", "openexr", "zlib", {optional = true})
+    add_includedirs("include/ImageMagick-7")
+
+    on_install("bsd", "linux", "macosx", function(package)
+        local configs = {"--with-utilities=no", "--with-x=no"}
+
+        if package:config("shared") then
+            table.insert(configs, "--enable-shared=yes")
+            table.insert(configs, "--enable-static=no")
+        else
+            table.insert(configs, "--enable-static=yes")
+            table.insert(configs, "--enable-shared=no")
+        end
```

Comment:
https://github.com/xmake-io/xmake-repo/blob/68fc348e5d650ae1ae4a054b2c0ae926defdd83e/packages/l/libisl/xmake.lua#L14-L15

Reply:
And please enable pic by default, like this

https://github.com/xmake-io/xmake-repo/blob/68fc348e5d650ae1ae4a054b2c0ae926defdd83e/packages/l/libisl/xmake.lua#L19-L21